app/boards/arm/nice_nano/nice_nano_v2.yaml

```diff
@@ -8,6 +8,7 @@ toolchain:
   - xtools
 supported:
   - adc
+  - encoders
```

Comment:
The nice!nano itself doesn't support encoders, no need to change this. A shield that the nice!nano plugs into may support encoders though! This also is a Zephyr specific file, which doesn't know anything about these encoders.